config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  get 'static_pages/home'
+  get 'static_pages/help'
```

Comment:
Ở các file .rb thì bạn đổi ' thành " hết nhé
```suggestion
  get "static_pages/home"
  get "static_pages/help"
```